Avoid literal "-n" appearing in files by using printf instead of echo -n.
--- setup/BLDUTIL3.i.orig	2018-07-14 10:20:25.000000000 -0500
+++ setup/BLDUTIL3.i	2018-07-17 12:25:37.000000000 -0500
@@ -1025,15 +1025,6 @@
 		case gbk_ide_snc:
 		case gbk_ide_dkp:
 		case gbk_ide_ccc:
-			WriteCStrToDestFile("echo");
-			if (! newline) {
-				WriteCStrToDestFile(" -n");
-			}
-			WriteCStrToDestFile(" \"");
-			ptext();
-			WriteCStrToDestFile("\" >");
-			WritePathArgInMakeCmnd(pto);
-			break;
 		case gbk_ide_mvc:
 		case gbk_ide_xcd:
 			WriteCStrToDestFile("printf \"");
--- setup/WRTEXTFL.i.orig	2018-07-14 10:46:30.000000000 -0500
+++ setup/WRTEXTFL.i	2018-07-17 12:16:05.000000000 -0500
@@ -192,7 +192,7 @@
 		WriteCStrToOutput(FileExt);
 		WriteCStrToOutput("\"");
 		WriteEolToOutput();
-		WriteLnCStrToOutput("echo -n > \"${DestFile}\"");
+		WriteLnCStrToOutput("printf \"\" > \"${DestFile}\"");
 		WriteEolToOutput();
 	}
 #endif
